File: PL_Final_Project/python/OOP/FindWords.py
class FindWords:
	def __init__(self):
		pass
    
	def process(self,data):
		split_words = []
		words = data.split("\\n")
		for line in words:
			words = line.split()
			for word in words:
				split_words.append(word)
		return split_words

File: PL_Final_Project/java/OOP/java/FindWords.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class FindWords implements IFindWords {

    /**
     * Constructor
     */
    public FindWords(){

    }

    @Override
    public List<String> process(String data) {
        List<String> splitWords = new ArrayList<>();
        String[] datas = data.split("\n");
        for (String line : datas) {
            splitWords.addAll(Arrays.asList(line.split(" ")));
        }
        return splitWords;
    }

}
